src/models/forecasts.jl

```diff
@@ -33,33 +30,30 @@ end
 """Container for forecasts and their metadata. Implementation detail that is not exported.
 Functions to access the data should go through the System."""
 struct SystemForecasts
-    data::_Forecasts
+    data::ForecastsByType
     initial_time::Dates.DateTime
     resolution::Dates.Period
     horizon::Int64
     interval::Dates.Period
 end
 
-"""Constructs SystemForecasts from the previous version of the data structure."""
-function SystemForecasts(old_forecasts::SystemForecastsDeprecated)
-    forecasts = _Forecasts()
+"""Constructs SystemForecasts from the flat vector of forecasts resulting from parsing."""
+function SystemForecasts(forecasts::Forecasts)
+    forecasts_by_type = ForecastsByType()
     initial_time = Dates.DateTime(Dates.Second(1))
     resolution = Dates.Period(Dates.Second(1))
```

Comment:
I had missed this from before, but why do we have the resolution here in Seconds?

Reply:
That's just to initialize the variables.  They are always overwritten.

Reply:
I'll change the values to 0.